app/views/shared/_hamburger.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="hamburger">
+  <div class="hamburger__content">
+    <header class="hamburger__content__header">
+      <h2 class="fs-l fw-bold flex-1 break-word lh-tight"><%= community_qualified_name.to_s %></h2>
+
+      <button class="crayons-btn crayons-btn--ghost crayons-btn--icon-rounded js-hamburger-trigger shrink-0">
+        <%= inline_svg_tag("x.svg", aria: true, class: "crayons-icon", title: "Close") %>
+      </button>
+    </header>
+
+    <div class="p-2">
+      <%= render partial: "shared/auth_widget" unless user_signed_in? %>
+
+      <nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" id="sidebar-nav" aria-label="Primary sidebar nav">
+        <a href="<%= root_path %>" class="crayons-link crayons-link--block">
+          <%= inline_svg_tag("twemoji/house.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Home") %>
+          Home
+        </a>
+        <% unless user_signed_in? %>
+          <a href="<%= sign_up_path %>" class="crayons-link crayons-link--block fw-bold">
+            <%= inline_svg_tag("twemoji/handshake.svg", aria: true, class: "crayons-icon crayons-icon--default", title: "Sign In/Up") %>
+            Sign In/Up
+          </a>
+        <% end %>
+        <div class="sidebar-navigation-links">
+          <% NavigationLink.all.ordered[0..4].each do |link| %>
```

Comment:
Can we store this `NavigationLink.all` in a variable and then access it that way? As is this will hit the database 3 times to fetch the same data.
```
<% links = NavigationLink.all %> 
<% links.ordered[0..4].each do |link| %>

# replace all NavigationLink.all calls with links then for this partial
```